api/src/test/java/org/pentaho/platform/api/genericfile/IGenericFileServiceTest.java

```diff
@@ -66,6 +69,12 @@ public boolean hasAccess( @NonNull GenericFilePath path, @NonNull EnumSet<Generi
     public boolean createFolder( @NonNull GenericFilePath path ) {
       throw new UnsupportedOperationException();
     }
+
+    //TODO implement
```

Comment:
Is there really anything to do here?

Reply:
no, I will follow the `throw new UnsupportedOperationException();` pattern